library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -1242,6 +1246,7 @@ void vtkF3DRenderer::ConfigureCheatSheet()
     cheatSheetText << " 3: Right View camera\n";
     cheatSheetText << " 7: Top View camera\n";
     cheatSheetText << " 9: Isometric View camera\n";
+    cheatSheetText << " 0: Toggle Orthogonal projection\n";
```

Comment:
@lknknm suggested 5 in the issue: https://github.com/f3d-app/f3d/issues/885

Any reason to change that ?

Reply:
simple wasnt aware of this ticket. The ticket I looked at was #1096.

Reply:
no worries, I should have pointed it out